tests/network/vmi_multus.go

```diff
@@ -685,13 +685,13 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 				tests.WaitAgentConnected(virtClient, agentVMI)
 
 				getOptions := &metav1.GetOptions{}
-				Eventually(func() bool {
+				Eventually(func() []v1.VirtualMachineInstanceNetworkInterface {
```

Comment:
ditto

Reply:
Replied in https://github.com/kubevirt/kubevirt/pull/7497#discussion_r841587815